features/202-session-management/design.md

```diff
@@ -55,34 +54,50 @@ to be revoked.
 Update session API allows updating name of session. Custom attributes can also
 be updated using master key.
 
+## Different kinds of session
 
-## Access / Refresh Token
+### OIDC session
 
-Each session would have an access token, and optionally a refresh token. Both
-type of tokens are opaque: token formats are unspecified and developer should
-not attempt to interpret the content of tokens.
+The sessions created by JSON API or OIDC flow have `client_id`.
 
-Access token would always have a lifetime; access token would be treated as
-invalid after its expiry.
+### OP session
+
+The sessions created by the web UI of Auth Gear does not have `client_id` because Auth Gear is acting as an OP.
+
+### Revoking sessions
+
+The user can revoke session of any kinds. However, revoking a OIDC session does not necessarily cause logout in the RP.
+
+When the RP is a third-party app that has its session management.
+When it receives a ID token from Auth Gear it creates its own session.
+Revoking the OIDC session has no direct effect on the third-party app session, unless both Auth Gear and the third-party app supports OIDC Session Management which offers single logout.
+
+If the RP is the same Skygear App, then revoking the OIDC session works seamlessly. It is because Skygear App does maintain a separate session.
```

Comment:
`Skygear App does maintain a separate session`?  Per our discussion, Skygear App should be the first party app and using the same session?

Reply:
Typo. Should be `does not maintain`